src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -985,8 +985,13 @@ void uct_rc_mlx5_ep_cleanup_qp(uct_ib_async_event_wait_t *wait_ctx)
                                                        uct_rc_mlx5_iface_common_t);
     uct_ib_mlx5_md_t *md              = ucs_derived_of(iface->super.super.super.md,
                                                        uct_ib_mlx5_md_t);
+    int count;
 
-    uct_rc_mlx5_iface_common_check_cqs_ci(iface, &iface->super.super);
+    count = uct_rc_mlx5_iface_commom_clean(&iface->cq[UCT_IB_DIR_RX],
+                                           &iface->rx.srq,
+                                           ep_cleanup_ctx->qp.qp_num);
+    iface->super.rx.srq.available += count;
+    uct_rc_mlx5_iface_common_update_cqs_ci(iface, &iface->super.super);
```

Comment:
1. would it be used in non-exp flow? can we disable?
2. did we already get last wqe reached at this point? need to assert it?

Reply:
2. this is either last_wqe callback or iface destroy

Reply:
2. does it make sense to add some assert the last wqe event arrived?

Reply:
2. by design we get here either in response to last_wqe or if we don't care of it anymore so assert won't work here,